manifests/z/ZAP/ZAP/2.13.0/ZAP.ZAP.locale.en-US.yaml

```
@@ -0,0 +1,30 @@
+PackageIdentifier: ZAP.ZAP
+PackageVersion: 2.13.0
+PackageLocale: en-US
+Publisher: ZAP
```

```suggestion
Publisher: OWASP ZAP
```
This should match control panel

---

Is this really necessary? 
ZAP is no longer part of OWASP. We will be releasing a new version, but dont have a date for it yet.

---

If you want to leave (and the Package Name) this as just `ZAP` you would need to add the `AppsAndFeaturesEntries` section in the Installer manifest so that winget can detect the version that is installed. However, if you set AppsAndFeaturesEntries for one version, it will also need to be set for all versions going forward (although the values will change to match what is written to the registry)

This would be -
```yaml
AppsAndFeaturesEntries:
- DisplayName: OWASP Zed Attack Proxy 2.13.0
  Publisher: OWASP ZAP
  ProductCode: OWASP ZAP
```

---

> if you set AppsAndFeaturesEntries for one version, it will also need to be set for all versions going forward

IIRC, that's only if we have DisplayVersion as it causes additional version mapping. I don't think the other ARP entries are an issue

---

> IIRC, that's only if we have DisplayVersion as it causes additional version mapping. I don't think the other ARP entries are an issue

Hmm, good point. I think you might be right, but still not a bad thing to carry it forward